spec/features/publish/courses/editing_course_outcome_spec.rb

```diff
@@ -74,7 +135,96 @@ def then_i_am_shown_the_correct_non_qts_options
     expect(publish_courses_outcome_edit_page.qualification_names).to contain_exactly('PGCE only (without QTS)', 'PGDE only (without QTS)')
   end
 
+  def and_i_choose_undergraduate_degree_with_qts
+    publish_courses_outcome_edit_page.undergraduate_degree_with_qts.choose
+  end
+
+  def and_i_choose_qts
+    publish_courses_outcome_edit_page.qts.choose
+    and_i_submit
+  end
+
+  def and_i_choose_the_fee_paying
+    choose 'Fee - no salary'
+    and_i_update
+  end
+
+  def and_i_choose_to_sponsor_a_student_visa
+    choose 'Yes'
+    and_i_update
+  end
+
+  def and_i_choose_part_time
+    uncheck 'Full time'
+    check 'Part time'
+    and_i_update
+  end
+
+  def and_the_tda_feature_flag_is_active
+    allow(Settings.features).to receive(:teacher_degree_apprenticeship).and_return(true)
+  end
+
   def provider
     @current_user.providers.first
   end
+
+  def given_i_am_authenticated_as_a_provider_user_in_the_next_cycle
+    next_cycle_providers = [build(:provider, :accredited_provider, :next_recruitment_cycle,
+                                  courses: [create(:course, :with_accrediting_provider, study_mode: 'part_time', funding_type: 'fee', can_sponsor_skilled_worker_visa: true)])]
+    @next_cycle_user = create(:user, providers: next_cycle_providers)
+    given_i_am_authenticated(user: @next_cycle_user)
+  end
+
+  def next_cycle_provider
+    @provider ||= @current_user.providers.first
+  end
+
+  def and_the_tda_feature_flag_is_active
+    allow(Settings.features).to receive(:teacher_degree_apprenticeship).and_return(true)
+  end
+
+  def and_there_is_a_qts_course_i_want_to_edit
+    given_a_course_exists(:resulting_in_qts, study_mode: 'part_time', funding_type: 'fee', can_sponsor_skilled_worker_visa: true)
+  end
+
+  def when_i_visit_the_course_outcome_page_in_the_next_cycle
+    publish_courses_outcome_edit_page.load(
+      provider_code: next_cycle_provider.provider_code, recruitment_cycle_year: next_cycle_provider.recruitment_cycle_year, course_code: next_cycle_provider.courses.first.course_code
+    )
+  end
+
+  def then_the_default_options_for_a_tda_course_should_be_applied
+    course.reload
+    expect(course.study_mode == 'full_time').to be(true)
+    expect(course.funding_type == 'apprenticeship').to be(true)
+    expect(course.can_sponsor_skilled_worker_visa == false).to be(true)
+    expect(course.can_sponsor_student_visa == false).to be(true)
```

Comment:
`expect(course.can_sponsor_student_visa).to be false`

Reply:
https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExb2ZsNGp3azlsY3JweGR4cWI2aDhodjVvZWRscDcyNWI4ODBkYngwdCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/COYGe9rZvfiaQ/giphy.gif


https://github.com/DFE-Digital/publish-teacher-training/pull/4223/commits/a963a2241d9626b4cf71a37fcf76c8823e4a40d6